cmd/occurrence.go

```diff
@@ -1,7 +1,7 @@
 package cmd
 
 type Occurrence struct {
-	kind    int    // or int? just needs to represent  {debug, log, warn, error}
+	kind    int    // represent  {debug, log, warn, error}
```

Comment:
may be using pesudo enum would be a good idea?

Reply:
This only holds an int which comes from the `loglevel.go` enum.